applications/connectivity_bridge/src/modules/Kconfig

```diff
@@ -24,6 +24,19 @@ source "subsys/logging/Kconfig.template.log_config"
 
 endif
 
+config BRIDGE_BULK_ENABLE
```

Comment:
Sure, we could also call it CMSIS_DAP, but that would be confusing, since we are not implementing that interface fully yet. The full name of this interface is CMSIS-DAP v2 BULK USB interface.